src/PyF/Internal/Meta.hs

```diff
@@ -0,0 +1,229 @@
+{-# LANGUAGE PackageImports #-}
+{-# LANGUAGE NamedFieldPuns #-}
+{-# LANGUAGE TemplateHaskellQuotes #-}
+
+module PyF.Internal.Meta (toExp, baseDynFlags, translateTHtoGHCExt) where
+
+import GHC.Hs.Expr as Expr
+import GHC.Hs.Extension as Ext
+import GHC.Hs.Types (HsWildCardBndrs (..), HsType (..))
+import GHC.Hs.Pat (HsRecFields (..))
+import GHC.Hs.Lit
+import qualified "template-haskell" Language.Haskell.TH.Syntax as TH
+import qualified "ghc-lib-parser" Language.Haskell.TH.Syntax as GhcTH
+import Language.Haskell.GhclibParserEx.Config (fakeLlvmConfig, fakeSettings)
+import qualified Data.ByteString as B
+import SrcLoc
+import Name
+import RdrName
+import FastString
+import Outputable (ppr, showSDocDebug)
+import BasicTypes (il_value, fl_value)
+import DynFlags (DynFlags, xopt_set, defaultDynFlags)
+
+toName :: Ext.IdP GhcPs -> TH.Name
+toName = TH.mkName . occNameString . rdrNameOcc
+
+toLit :: HsLit GhcPs -> TH.Lit
+toLit (HsChar _ c)        = TH.CharL c
```

Comment:
Could you write a more detailed `_` wildcard, such as `_sourceLoc` (in this context), so it is easier to understand what is ignored.